include/RAJA/policy/openmp/synchronize.hpp

```diff
@@ -0,0 +1,44 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-18, Lawrence Livermore National Security, LLC.
+//
+// Produced at the Lawrence Livermore National Laboratory
+//
+// LLNL-CODE-689114
+//
+// All rights reserved.
+//
+// This file is part of RAJA.
+//
+// For details about use and distribution, please read RAJA/LICENSE.
+//
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_synchronize_openmp_HPP
+#define RAJA_synchronize_openmp_HPP
+
+namespace RAJA
+{
+
+namespace policy
+{
+
+namespace omp
+{
+
+/*!
+ * \brief Synchronize all OpenMP threads and tasks.
+ */
+RAJA_INLINE
+void
+synchronize_impl(const omp_synchronize&)
+{
+#pragma omp barrier
```

Comment:
I just double checked this, the barrier is sufficient to wait for all tasks, so the taskwait can come out unless we want a slightly smaller hammer.

Reply:
Great, the `taskwait` fails on AppVeyor too.